config/application.rb

```diff
@@ -24,6 +24,7 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
     config.autoloader = :classic
+    # FIXME Autoloader is only set to classic while restructuring work is undertaken.
```

Comment:
This comment should be moved to the relevant commit.

Reply:
Same as above, I'll move it's own commit.